sdk/search/azure-search-documents/swagger/readme.md

```diff
@@ -19,7 +19,7 @@ To see additional help and options, run:
 Fork and clone https://github.com/Azure/autorest.java 
 git checkout v4
 git submodule update --init --recursive
-mvn package -Dlocal
+mvn package -Dlocal -DskipTests
```

Comment:
Looks like a typo here?

Reply:
The test needs to setup local server to run. So I skip tests which is enough for building the package.